python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -148,7 +149,11 @@ def setDefaults(self):
         self.measurement.algorithms.names.add('base_PeakLikelihoodFlux')
         self.measurement.plugins.names |= ['ext_trailedSources_Naive',
                                            'base_LocalPhotoCalib',
-                                           'base_LocalWcs']
+                                           'base_LocalWcs',
+                                           'ext_shapeHSM_HsmSourceMoments',
+                                           'ext_shapeHSM_HsmPsfMoments']
```

Comment:
The following would be more forward-compatible with future changes:
```suggestion
                                           'ext_shapeHSM_HsmPsfMoments',
                                           ]
```